app/controllers/api/v0/users_controller.rb

```diff
@@ -23,6 +25,8 @@ def show
                 end
       end
 
+      def me; end
```

Comment:
I've noticed that `users/me.json.jbuilder` is identical to `users/show.json.jbuilder`, hence, I'm not totally sure why we need an authenticated `me` endpoint since we have (the albeit unintuitive) https://dev.to/api/users/by_username?url=rhymes

If you think this one is needed anyway, you can render the same file as the show endpoint like this:

```ruby
def me
  render "show"
end
```

Reply:
I thought that an inflexible endpoint that accepts no param would be a safer move to return a response filled with more private information, made with an authenticated request.

But having it's own view does seem a bit redundant. I can modify `show.json.jbuilder` with some `if` statement. What do you think?

Reply:
Sorry if I wasn't clearer: I agree with the idea of the authenticated `/me` returning private info, it's just that since you asked for a review I thought it was finished, hence my comment about redudancy :D

Reply:
All good!